### DISCUSSION TITLE
Frigate 14 detection "area px" and "ratio"

### DISCUSSION THREAD

**Discussion by philippedegeer** [blakeblackshear/frigate#12107] [General]
I have recently transitioned to Frigate 14beta3 and navigated quite a bit through the UI and the documentation.
I have not found the detection "area px" and "ratio" data that Frigate 13 showed, together with zone time entered/left. Those were very helpful to tune detection filtering.
Are these metrics no longer included ?

**Comment by hawkeye217** [comment]
This does not currently exist in 0.14 beta. There is a feature request here: https://github.com/blakeblackshear/frigate/issues/11399

The debug pane in Settings will show area and ratio for active objects, but not historical objects. 

Feel free to leave a thumbs up on the feature request.

**Reply by philippedegeer** [reply]
Thanks Josh, just did it.
